### DISCUSSION TITLE
Thanks. Github. Am new year hoping to make the best of this tools

### DISCUSSION THREAD

**Discussion by naphtha** [community/community#17743] [GitHub Education]
Please how do I get the promo code for a digital ocean account?

**Comment by samus-aran** [comment]
Once verified you should be able to enable any of the offers by visiting the benefits page and navigating to the Digital Ocean offer (I believe it should have a **Details** button and/or **"Request your offer code to get access"**). [DigitalOcean](https://www.digitalocean.com/github-students) also assist by this unique page to help students get started.
